apps/avatax/README.md

```diff
@@ -0,0 +1,23 @@
+![Saleor Taxes App](https://user-images.githubusercontent.com/249912/71523206-4e45f800-28c8-11ea-84ba-345a9bfc998a.png)
```

Comment:
```suggestion
![Saleor AvaTax App](https://user-images.githubusercontent.com/249912/71523206-4e45f800-28c8-11ea-84ba-345a9bfc998a.png)
```

Also, there are other occurrences of the phrase "taxes app" in the `avatax` code.